pkg/awsclient/elb_helper.go

```diff
@@ -22,8 +21,15 @@ type AWSLoadBalancer struct {
 // ELB should attend, as well as the listener port.
 // The port is used for the instance port and load balancer port
 // Return is the (FQDN) DNS name from Amazon, and error, if any.
-func (c *awsClient) CreateClassicELB(elbName string, subnets []string, listenerPort int64) (*AWSLoadBalancer, error) {
-	fmt.Printf("  * CreateClassicELB(%s,%s,%d)\n", elbName, subnets, listenerPort)
+func (c *AwsClient) CreateClassicELB(elbName string, subnets []string, listenerPort int64, tagList map[string]string) (*AWSLoadBalancer, error) {
+	tags := make([]*elb.Tag, 0)
```

Comment:
Could this bit of logic be collapsed into a shared function? I see the same code in ec2_helper.go `ApplyTagsToResources`